README.md

```diff
@@ -7,34 +7,29 @@
 </h3>
 
 
-This repo implements a document pre-processing pipeline for SEC filings. Currently, the pipeline
-covers extracting narrative text from a user-specified section in 10-K, 10-Q, and S-1 filings.
-The API is hosted at `https://api.unstructured.io`.
+This repo implements a document pre-processing pipeline for SEC filings. Currently, the pipeline is capable of extracting narrative text from user-specified sections in 10-K, 10-Q, and S-1 filings.
 
-## Getting Started
+The API is hosted at `https://api.unstructured.io`.
 
-To view the most common `Makefile` targets, run `make`. The `Makefile` targets include several
-options for installing and/or running the pipeline.
+## Developer Quick Start
 
-To install all of the dependencies for the pipeline, run `make install`. The pipeline is intended
-to be run from the base directory of this repo. If you want to run the pipeline from another
-directory, ensure the base directory of the repo is added to your Python path. You can do that
-by running `export PYTHONPATH=${PWD}:${PYTHONPATH}` from this directory.
+* Using `pyenv` to manage virtualenv's is recommended
+	* Mac install instructions:
+		*  `pyenv brew install pyenv-virtualenv`
```

Comment:
It looks like something is missing from this.

Reply:
Not sure what you mean. This is what the rendering looks like.

<img width="741" alt="image" src="https://user-images.githubusercontent.com/28578599/200231139-48ea9bdf-70a6-4065-a6b8-e4577fdd6080.png">

Reply:
Verbose dev instructions is a TODO for https://github.com/Unstructured-IO/community

Reply:
Sorry should have been more specific.

I'm pretty sure `pyenv brew install pyenv-virtualenv` won't work. It's probably meant to be `brew install pyenv pyenv-virtualenv` or broken up into 2 lines if `brew` doesn't support that syntax.

Reply:
fixed the brew command, though